From 9006c08eed3961109c79ff6fa69b0ffcb06e4289 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20H=C3=B8ydahl?= <jan.git@cominvent.com>
Date: Tue, 10 Mar 2015 10:59:24 +0100
Subject: [PATCH] Fixed wrong url in README and wrong version numbers in
 BUILDING

---
 BUILDING | 14 +++++++-------
 README   |  2 +-
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/BUILDING b/BUILDING
index 260e8d2..7f7ca8e 100644
--- a/BUILDING
+++ b/BUILDING
@@ -15,20 +15,20 @@
  * limitations under the License.
  */
 
-Building Apache ServiceMix 5
+Building Apache ServiceMix 6
 ============================
 
 Initial Setup
 -------------
 
-1) Install J2SE 5.0 SDK, which can be downloaded from 
-   http://java.sun.com/j2se/1.5.0/download.jsp. Use version of 1.5.0_06 JDK.
+1) Install J2SE 8.0 SDK, which can be downloaded from
+   http://www.oracle.com/technetwork/java/javase/downloads/index.html. Use latest version.
 
 2) Make sure that your JAVA_HOME environment variable is set to the newly installed 
    JDK location, and that your PATH includes %JAVA_HOME%\bin (windows) or 
    $JAVA_HOME$/bin (unix).
 
-3) Install Maven 3.0.5, which can be downloaded from 
+3) Install Maven 3.2.5, which can be downloaded from
    http://maven.apache.org/download.html. Make sure that your PATH includes 
    the MVN_HOME/bin directory. 
 
@@ -36,18 +36,18 @@ Initial Setup
 Building
 --------
 
-1) Change to the top level directory of Apache ServiceMix 5 source distribution.
+1) Change to the top level directory of Apache ServiceMix source distribution.
 
 2) Run
          $> mvn
          
-   This will compile Apache ServiceMix 4 and run all of the tests in the 
+   This will compile Apache ServiceMix and run all of the tests in the
    Apache ServiceMix Kernel source distribution. 
    
    Alternatively, you can run
          $> mvn -Pfastinstall
          
-   This will compile Apache ServiceMix 4 without running the tests and takes less
+   This will compile Apache ServiceMix without running the tests and takes less
    time to build.
    
    Depending on the load of remote Maven repositories, you may have 
diff --git a/README b/README
index 8657be6..d15149c 100644
--- a/README
+++ b/README
@@ -43,7 +43,7 @@ Alternatively, you can also find out how to get started here:
     http://servicemix.apache.org/docs/6.x/
 
 If you need more help try talking to us on our mailing lists
-    http://servicemix.apache.org/mailing-lists.html
+    http://servicemix.apache.org/community/mailing-lists.html
 
 If you find any issues with ServiceMix 6, please submit reports
 with JIRA here:
